server/handler/contributor.go

```
@@ -0,0 +1,309 @@
+		}
+		if dbUser.Login == "" {
+			log.Printf("[Contributor Handler] No user found in DB for login: %s", login)
+			http.Error(w, "User not found", http.StatusNotFound)
+			return
+		}
+
+		// Check cache
+		if val, found := userCache.Get(login); found {
+			if cached, ok := val.(cachedUser); ok && time.Since(cached.Timestamp) < cacheExpiry {
+				log.Printf("[Contributor Handler] Cache hit for user: %s", login)
+				json.NewEncoder(w).Encode(cached.Data)
```

golangci-lint (1.64.8)

Error return value of `(*encoding/json.Encoder).Encode` is not checked

(errcheck)